plugins/mcreator-localization/help/default/block/placing_condition.md

```diff
@@ -0,0 +1,4 @@
+This condition determines whether this block can be placed in a specific location. If the position is
+no longer valid, the block will be broken.

```

Comment:
> If the position is no longer valid, the block will be broken.
 
I have added condition to place only if there is no rain and when I toggle rain, existing blocks don't get removed so this is either bug, or this description needs rewording.

Reply:
It's just rewording, the condition is checked after the block receives a block update (same thing as plants, floating grass doesn't break until you place something next to it)